package.json

```diff
@@ -20,6 +20,7 @@
     "query-string": "^6.8.3",
     "react": "16.8.3",
     "react-native": "^0.59.9",
+    "react-native-camera": "^3.14.0",
```

Comment:
Remove all `^` character in every package version to lock current working version

Reply:
ok